common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/microsoftsts/MicrosoftStsOAuth2Configuration.java

```diff
@@ -52,22 +52,15 @@
      * @return URL the authorization endpoint
      */
     public URL getAuthorizationEndpoint() {
-        final OpenIdProviderConfiguration openIdConfig = getOpenIdWellKnownConfigForAuthority();
-        if (openIdConfig != null) {
-            return getEndpointUrlFromAuthority(openIdConfig.getAuthorizationEndpoint());
-        }
         return getEndpointUrlFromRootAndSuffix(getAuthorityUrl(), FALLBACK_AUTHORIZE_ENDPOINT_SUFFIX);
```

Comment:
Rename to `AUTHORIZE_ENDPOINT_SUFFIX` ? and similar below

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1015/commits/161957ecc585b2fb06601d6ff19c96df7f3d494c